### PR TITLE
Convert create-your-own-core-image to Markdown

### LINKED ISSUE #267: convert - # Create your own Core image tutorial
172iM3jTkg2os8HgqtSpVug3IR5pjqS2zz607hTi3KVg

### PR DESCRIPTION
**Pull request by evilnick**
Fixes #267

### CODE REVIEW

File: tutorials/snap/create-your-own-core-image.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+
+
+# Create your own Core image
+
+ 
+## Overview
+Duration: 1:00
+
+We are going to build an image for the [Intel Joule] device. Including extra snaps compared to the default provided Intel Joule Ubuntu Core image. The same techniques can be replicated on any other devices.
+We are going to create your own authority keys, making the snap store aware of them, create and sign your model assertion before building your image.
```
Comment (degville) [inline tutorials/snap/create-your-own-core-image.md]: authority keys and model assertion are bold in the original.

File: tutorials/snap/create-your-own-core-image.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+---
+
+
+# Create your own Core image
+
+ 
+## Overview
+Duration: 1:00
+
+We are going to build an image for the [Intel Joule] device. Including extra snaps compared to the default provided Intel Joule Ubuntu Core image. The same techniques can be replicated on any other devices.
```
Comment (degville) [inline tutorials/snap/create-your-own-core-image.md]: These paras need blank lines between them to render as before.

File: tutorials/snap/create-your-own-core-image.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+
+## Dissection of an Ubuntu Core image
+Duration: 4:00
+
+### What your image is made of
+
+![IMAGE](https://assets.ubuntu.com/v1/e997c521-snap-content.png)
+
+
+If you simply run `snap list` on your Ubuntu Core image, you will see that your image is already composed of many snap. For instance, on our Intel Joule:
```
Comment (degville) [inline tutorials/snap/create-your-own-core-image.md]: snap-list was in double quotes before.

File: tutorials/snap/create-your-own-core-image.md [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+
+
+```bash
+$ date -Iseconds --utc
+2017-01-06T15:54:34+00:00
+```
+
+
+positive
+: Note that this timestamp should be posterior to the signing key creation.
```
Comment (degville) [inline tutorials/snap/create-your-own-core-image.md]: This wasn't an admonition before, but it still looks ok.

File: tutorials/snap/create-your-own-core-image.md (hunk 3 above)
Comment (caldav) [inline tutorials/snap/create-your-own-core-image.md]: @evilnick yours is the correct format

File: tutorials/snap/create-your-own-core-image.md (hunk 2 above)
Comment (evilnick) [inline tutorials/snap/create-your-own-core-image.md]: :+1:

File: tutorials/snap/create-your-own-core-image.md (hunk 1 above)
Comment (evilnick) [inline tutorials/snap/create-your-own-core-image.md]: :+1:

File: tutorials/snap/create-your-own-core-image.md (hunk 3 above)
Comment (evilnick) [inline tutorials/snap/create-your-own-core-image.md]: Yes, I naughtily decided to correct that bit as I went along :smile:

File: tutorials/snap/create-your-own-core-image.md (hunk 4 above)
Comment (evilnick) [inline tutorials/snap/create-your-own-core-image.md]: I think it works better this way

**Pull request merged**